app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -82,43 +78,43 @@ class MainFragment : Fragment() {
         recyclerView = view!!.findViewById<RecyclerView>(R.id.rec_currentList)
         var userName = view!!.findViewById<TextView>(R.id.username)
         var userPoints = view!!.findViewById<TextView>(R.id.points)
-        timerTextView  = view!!.findViewById(R.id.tv_mainListTimer)
+        timerTextView = view!!.findViewById(R.id.tv_mainListTimer)
         var listTitleLabel = view!!.findViewById<TextView>(R.id.tv_mainListTitle)
         //userName.text = viewModel.getUserName()
         //userPoints.text = viewModel.getUserPoints()
 
 
-    //    viewModel.firestoreService.resetExpirationDateOnAllLists(countdownTime.toInt())
+        //    viewModel.firestoreService.resetExpirationDateOnAllLists(countdownTime.toInt())
 
 
-        recyclerView.layoutManager =  LinearLayoutManager(this.context)
+        recyclerView.layoutManager = LinearLayoutManager(this.context)
         (recyclerView.getItemAnimator() as SimpleItemAnimator).supportsChangeAnimations = false
 
         viewModel.loadNextList(false)
 
-     /*   var listItemsToAdd = ArrayList<ListItem>()
-        listItemsToAdd.add(ListItem(0,"Cinnamon Toast Crunch","",0))
-        listItemsToAdd.add(ListItem(1,"Lucky Charms","",0))
-        listItemsToAdd.add(ListItem(2,"Froot Loops","",0))
-        listItemsToAdd.add(ListItem(3,"Cheerios","",0))
-        listItemsToAdd.add(ListItem(4,"Frosted Flakes","",0))
-        listItemsToAdd.add(ListItem(5,"Honeycomb","",0))
-        listItemsToAdd.add(ListItem(6,"Cap'n Crunch","",0))
-        listItemsToAdd.add(ListItem(6,"Reese's Puffs","",0))
-
-        var list: TopTenList = TopTenList(
-            1,
-            "Top Ten Cereals",
-            "A list of favorite cereals",
-            false,
-            "Food",
-            Date(),
-            Date()
-        )
-        list.listItems = listItemsToAdd
-        viewModel.firestoreService.writeListToDatabase(list)
-
-      */
+        /*   var listItemsToAdd = ArrayList<ListItem>()
+           listItemsToAdd.add(ListItem(0,"Cinnamon Toast Crunch","",0))
+           listItemsToAdd.add(ListItem(1,"Lucky Charms","",0))
+           listItemsToAdd.add(ListItem(2,"Froot Loops","",0))
+           listItemsToAdd.add(ListItem(3,"Cheerios","",0))
+           listItemsToAdd.add(ListItem(4,"Frosted Flakes","",0))
+           listItemsToAdd.add(ListItem(5,"Honeycomb","",0))
+           listItemsToAdd.add(ListItem(6,"Cap'n Crunch","",0))
+           listItemsToAdd.add(ListItem(6,"Reese's Puffs","",0))
+
+           var list: TopTenList = TopTenList(
+               1,
+               "Top Ten Cereals",
+               "A list of favorite cereals",
+               false,
+               "Food",
+               Date(),
+               Date()
+           )
+           list.listItems = listItemsToAdd
+           viewModel.firestoreService.writeListToDatabase(list)
+
+         */
```

Comment:
It's commented code.  Delete before merging.

 Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.